src/components/Medicine/MedicationAdministration/MedicineAdminForm.tsx

```
@@ -0,0 +1,400 @@
+          }
+        : {
+            occurrence_period_end: date.toISOString(),
+          }),
+    });
+  };
+
+  const formatTime = (date: string | undefined) => {
+    if (!date) return "";
+    try {
+      const dateObj = new Date(date);
+      if (isNaN(dateObj.getTime())) return "";
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)